features/step_definitions/components/research_evaluation_steps.rb

```diff
@@ -0,0 +1,16 @@
+Given('I entered a page which includes the R&E component') do
+  step %(I entered into the Lifestage page "Young Adults")
+end
+
+Then('I should see {string} links for {string}') do |number, title, table|
+  items = current_page.research_and_evaluation_boxes.select do |box|
+    box.title.text == title
+  end.first.research_rows
```

Comment:
Can you change Array#select.first for Array#find? Also can we move this whole thing to a method on the Page object? Something like I did on pacs: https://github.com/moneyadviceservice/pacs/blob/master/features/support/ui/results_page.rb#L14